workflows/metapep.nf

```diff
@@ -309,7 +269,7 @@ workflow METAPEP {
     GENERATE_PEPTIDES.out.ch_peptides,
     GENERATE_PEPTIDES.out.ch_proteins_peptides,
     GENERATE_PROTEIN_AND_ENTITY_IDS.out.ch_entities_proteins,
-    FINALIZE_MICROBIOME_ENTITIES.out.ch_microbiomes_entities,
+    GENERATE_PROTEIN_AND_ENTITY_IDS.out.ch_microbiomes_entities_noweights,
```

Comment:
`ch_microbiomes_entities_noweight` does this channel name still make sense? It was originally name in this way to make clear that it does not yet contain the weights, but there is no channel `ch_microbiomes_entities` anymore containing weights, right?

Reply:
True, there is only a table that connects conditions to weights but I didn't rename it yet, so it is still called ch_microbiomes_entities because I used the terms conditions and microbiomes kind of interchangeably but of course that's confusing, maybe we can talk about a clear definition for them